modules/virt-customizing-storage-profile.adoc

```diff
@@ -19,21 +22,21 @@ If you create a data volume and omit YAML attributes and these attributes are no
 ====
 
 .Procedure
-. Edit the storage profile. In this example, the provisioner is not recognized by the CDI:
+. Edit the storage profile. In this example, the provisioner is not recognized by CDI:
 +
-[source,terminal]
+[source,YAML]
 ----
 $ oc edit -n openshift-cnv storageprofile <storage_class>
 ----
 +
-.Example output
-[source,terminal]
+.Example storage profile
+[source,YAML]
```

Comment:
Not sure why, but the syntax highlighting doesn't look right in the preview build. It might be because YAML is capitalized. Try switching to `[source,yaml]`

Reply:
Done.